content/release-0.24.0.md

```diff
@@ -1,3 +1,142 @@
+
+In the past, you'd hear things like "Use `.values` to extract the NumPy array
+from a Series or DataFrame." If it were a good resource, they'd tell you that's
+not *entirely* true, since there are some exceptions. I'd like to delve into
+those exceptions now.
+
+`.values` overloaded two purposes:
+
+1. Extracting the array backing a Series, Index, or DataFrame
+2. Converting the Series, Index, or DataFrame to a NumPy array
+
+As we saw above, the "array" backing a Series or Index might not be a NumPy
+array, it may an extension array (from pandas or a third-party library). For
+`Categorical`,
+
+```python
+>>> cat = pd.Categorical(['a', 'b', 'a'], categories=['a', 'b', 'c'])
+>>> ser = pd.Series(cat)
+>>> ser
+0    a
+1    b
+2    a
+dtype: category
+Categories (3, object): [a, b, c]
+
+>>> ser.values
+[a, b, a]
+Categories (3, object): [a, b, c]
+>>> ser = pd.Series(
+```
+
+So, `.values` isn't always a NumPy array. Even for dtypes that do return a NumPy
+array, it may not be what you want. For period-dtype data, `.values` returns a
+NumPy array of `Period` objects, which is expensive to create. For
+timezone-aware data, `.values` converts to UTC and *drops* the timezone info.
+These kind of surprises stem from trying to shoehorn these extension arrays into
+a NumPy array, when the entire point of an extension array is for representing
+data NumPy can't natively represent.
+
+We've split the two roles previously occupied by `.values` into two separate
+methods.
+
+1. Use `.array` to get a zero-copy reference to the underlying data
+2. Use `.to_numpy()` to get a (potentially expensive, lossy) NumPy array of the
+   data.
+  
+So with our Categorical example,
+
+```python
+>>> ser.array
+[a, b, a]
+Categories (3, object): [a, b, c]
+
+>>> ser.to_numpy()
+array(['a', 'b', 'a'], dtype=object)
+```
+  
+`.array` will *always* be a an ExtensionArray, and is always a zero-copy
+reference back to the data.
+
+`.to_numpy()` is *always* a NumPy array, so you can reliably call
+ndarray-specific methods on it.
+ 
+
+To summarize,
+
+- Use `.to_numpy()` if you absolutely need a NumPy array, and are willing to
+  copy or coerce data, or lose type information in the process
+- Use `.array` if you need a zero-copy reference to the underlying data.
+
+## Possible Future Paths
+
+I'd like to emphasize that this is an *interface*, and not a concrete array
+implementation. We are *not* reimplementing NumPy here in pandas. Rather, this
+is a way to take any array-like data structure (one or more NumPy arrays, an
+Apache Arrow array, a CuPy array) and place it inside a DataFrame. I think
+getting pandas out of the array business, and instead thinking about
+higher-level tabular data things, is a health development for the project.
+
+Extension arrays provide some interesting opportunities for pandas. 0.24.0
+introduced `IntegerArray`, a nullable-integer dtype, solving one of
+pandas-longest running feature requests.
```

Comment:
This is a big enough deal that you might talk about it in a section in the main text; some people might feel like you're buring the lede =)